packages/advanced-logic/specs/payment-network-any-to-erc20-proxy-0.1.0.md

```diff
@@ -60,6 +60,7 @@ The `TransferWithReferenceAndFee` event is emitted when the tokens are transfere
 | Mainnet                    | 0xe72Ecea44b6d8B2b3cf5171214D9730E86213cA2 |
 | Matic                      | 0xf0f49873C50765239F6f9534Ba13c4fe16eD5f2E |
 | Ethereum Testnet - Rinkeby | 0x78334ed20da456e89cd7e5a90de429d705f5bc88 |
+| Ethereum Testnet - Goerli  | TODO ..................................... |
```

Comment:
we have the address don't we?

Reply:
I don't think we have the contract address yet for this contract in particular on goerli